addons/TBMod_main/configs/ExtendedEventHandlers.hpp

```diff
@@ -50,4 +51,4 @@ class Extended_Init_EventHandlers
             serverInit = "params ['_vehicle']; {if !(_x isEqualTo _vehicle) then {deleteVehicle _x}; false} count (entities (typeOf _vehicle)); if (isNil 'TB_main_fobs') then {TB_main_fobs = []}; private _displayname = getText (configFile >> 'CfgVehicles' >> typeOf _vehicle >> 'displayName'); TB_main_fobs set [getNumber (configFile >> 'CfgVehicles' >> typeOf _vehicle >> 'fobNumber'), [_displayname, _vehicle]];  publicVariable 'TB_main_fobs'; private _marker = createMarker [_displayname, getPos _vehicle]; _marker setMarkerText _displayname; _marker setMarkerType 'hd_flag'; (format ['%1 wurde errichtet!', _displayname]) remoteExecCall ['systemChat']";
         };
     };
-};
\ No newline at end of file
+};
```

Comment:
Würde mal in den ungeschriebenen CodingConventions festhalten immer eine leere Zeile am Ende ☺